docs/quickstart/security.md

```diff
@@ -77,9 +77,9 @@ root@syscont:/# cat /proc/self/gid_map
          0  268994208      65536
 ```
 
-This means that user-IDs in the range [0:65535] inside the container are mapped
+This means that user-IDs in the range \[0:65535] inside the container are mapped
```

Comment:
Added by the markdown lint tool (remark); prefer to leave it there since I always run that tool on each file I edit. The "\" does not show up in the rendered document.